src/settings/Settings.ts

```diff
@@ -258,7 +258,7 @@ export const SETTINGS: {[setting: string]: ISetting} = {
     "MessageComposerInput.showStickersButton": {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         displayName: _td('Show stickers button'),
-        default: true,
+        default: false,
```

Comment:
You have changed the default element behaviour, instead please add the config item and set it to false in config-communicate.json and aether-host-files config.json